react/features/invite/middleware.web.js

```diff
@@ -42,3 +44,22 @@ function _beginAddPeople({ dispatch }, next, action) {
 
     return result;
 }
+
+/**
+ * Notifies the feature invite that the action {@link STOP_ADD_PEOPLE} is being
+ * dispatched within a specific redux {@code store}.
+ *
+ * @param {Store} store - The redux store in which the specified {@code action}
+ * is being dispatched.
+ * @param {Dispatch} next - The redux {@code dispatch} function to dispatch the
+ * specified {@code action} to the specified {@code store}.
+ * @param {Action} action - The redux action {@code STOP_ADD_PEOPLE} which is
+ * being dispatched in the specified {@code store}.
+ * @private
+ * @returns {*} The value returned by {@code next(action)}.
+ */
+function _stopAddPeople({ dispatch }, next, action) {
```

Comment:
I think this should be better something like `hideAddPeopleDialog` or something similar, as well as the action name